src/Creitive/Breadcrumbs/BreadcrumbsServiceProvider.php

```diff
@@ -11,7 +13,14 @@ class BreadcrumbsServiceProvider extends ServiceProvider {
 	 */
 	public function boot()
 	{
-		$this->package('creitive/breadcrumbs');
+        if ($this->isLegacyLaravel() || $this->isOldLaravel())
+        {
+		    $this->package('creitive/breadcrumbs');
+        }
+
+        $loader = \Illuminate\Foundation\AliasLoader::getInstance();
+        $loader->alias('Breadcrumbs', 'Creitive\Breadcrumbs\Facades\Breadcrumbs');
```

Comment:
My opinion is that packages shouldn't register facade aliases, as some users might not want them, and the package should not force this on them. Adding a facade alias is a one-line addition to the app config file, so it's not like it requires a lot of work from the users.